DataFormats/Scouting/interface/ScoutingMuon.h

```diff
@@ -7,63 +7,100 @@
+  float chi2() const { return chi2_; }
+  int ndof() const { return ndof_; }
+  int charge() const { return charge_; }
+  float dxy() const { return dxy_; }
+  float dz() const { return dz_; }
+  int nValidMuonHits() const { return nValidMuonHits_; }
+  int nValidPixelHits() const { return nValidPixelHits_; }
+  int nMatchedStations() const { return nMatchedStations_; }
+  int nTrackerLayersWithMeasurement() const { return nTrackerLayersWithMeasurement_; }
+  int type() const { return type_; }
+  bool isGlobalMuon() const { return type_ & 1<<1; }
+  bool isTrackerMuon() const { return type_ & 1<<2; }
+  int nValidStripHits() const { return nValidStripHits_; }
+  float trk_qoverp() const { return trk_qoverp_; }
+  float trk_lambda() const { return trk_lambda_; }
+  float trk_pt() const { return trk_pt_; }
+  float trk_phi() const { return trk_phi_; }
+  float trk_eta() const { return trk_eta_; }
+  float dxyError() const { return dxyError_; }
+  float dzError() const { return dzError_; }
+  float trk_qoverpError() const { return trk_qoverpError_; }
+  float trk_lambdaError() const { return trk_lambdaError_; }
+  float trk_phiError() const { return trk_phiError_; }
+  float trk_dsz() const { return trk_dsz_; }
+  float trk_dszError() const { return trk_dszError_; }
+  std::vector<int> vtxIndx() const { return vtxIndx_; }
 
-        //accessor functions
-        float pt() const { return pt_; }
-        float eta() const { return eta_; }
-        float phi() const { return phi_; }
-        float m() const { return m_; }
-        float ecalIso() const { return ecalIso_; }
-        float hcalIso() const { return hcalIso_; }
-        float trackIso() const { return trackIso_; }
-        float chi2() const { return chi2_; }
-        float ndof() const { return ndof_; }
-        int charge() const { return charge_; }
-        float dxy() const { return dxy_; }
-        float dz() const { return dz_; }
-        int nValidMuonHits() const { return nValidMuonHits_; }
-        int nValidPixelHits() const { return nValidPixelHits_; }
-        int nMatchedStations() const { return nMatchedStations_; }
-        int nTrackerLayersWithMeasurement() const { return nTrackerLayersWithMeasurement_; }
-        int type() const { return type_; }
-        bool isGlobalMuon() const { return type_ & 1<<1; }
-        bool isTrackerMuon() const { return type_ & 1<<2; }
-
-    private:
-        float pt_;
-        float eta_;
-        float phi_;
-        float m_;
-        float ecalIso_;
-        float hcalIso_;
-        float trackIso_;
-        float chi2_;
-        float ndof_;
-        int charge_;
-        float dxy_;
-        float dz_;
-        int nValidMuonHits_;
-        int nValidPixelHits_;
-        int nMatchedStations_;
-        int nTrackerLayersWithMeasurement_;
-        int type_;
+ private:
+  float pt_;
+  float eta_;
+  float phi_;
+  float m_;
+  float ecalIso_;
+  float hcalIso_;
+  float trackIso_;
+  float chi2_;
+  int ndof_;
```

Comment:
This appears to be the only possibly 'breaking' change, `ndof_` used to be `double` but is now an `int`.